linux/ibus-keyman/ibus-keyman.spec.in

```diff
@@ -0,0 +1,86 @@
+%define _prefix	/usr
+
+%define is_mandrake %(test -e /etc/mandrake-release && echo 1 || echo 0)
+%define is_mandriva %(test -e /etc/mandriva-release && echo 1 || echo 0)
+%define is_suse %(test -e /etc/SuSE-release && echo 1 || echo 0)
+%define is_opensuse %(grep open /etc/SuSE-release 2> /dev/null 1>/dev/null && echo 1 || echo 0)
+%define is_fedora %(test -e /etc/fedora-release && echo 1 || echo 0)
+
+%if %is_mandrake
+%define dist mandrake
+%define disttag mdk
+%endif
+%if %is_mandriva
+%define dist mandriva
+%define disttag mdk
+%endif
+%if %is_suse
+%define dist suse
+%define disttag suse
+%define kde_path /opt/kde3
+%endif
+%if %is_opensuse
+%define dist openSUSE
+%define disttag openSUSE
+%define kde_path /opt/kde3
+%endif
+%if %is_fedora
+%define dist fedora
+%define disttag fc
+%endif
+
+%define distver %(release="`rpm -q --queryformat='%{VERSION}' %{dist}-release 2> /dev/null | tr . : | sed s/://g`" ; if test $? != 0 ; then release="" ; fi ; echo "$release")
+
+Name:       ibus-kmfl
```

Comment:
This needs either updating to refer to ibus-keyman and check it still works on RPM distros or removing and allow the packagers there to handle this.

Reply:
> 
> 
> This needs either updating to refer to ibus-keyman and check it still works on RPM distros or removing and allow the packagers there to handle this.

removed. it can be retrieved from git if any packager wants to try it.